tests/enginetests/operator_string_test.go

```diff
@@ -467,3 +467,63 @@ func TestIndexerString(t *testing.T) {
 		})
 	}
 }
+
+func TestEndsWith(t *testing.T) {
+	tests := []struct {
+		name       string
+		cql        string
+		wantModel  model.IExpression
+		wantResult result.Value
+	}{
+		{
+			name: "EndsWithTrue",
+			cql:  "EndsWith('Chris Schuler is the man!!', 'n!!')",
```

Comment:
Looks like you might've copied tests from the spec tests in here, that's not generally what we do with these. Would you mind changing the strings in here to be something shorter to prove the case? perhaps something simple like 'apple' ends with 'ple'?

Additionally please add the following case:
only the left arg is null,
left arg is an empty string but right arg isnt,
right arg is an empty string but left isnt,
both args are empty strings